app/routes/dashboards/dashboard.js

```diff
@@ -231,294 +221,45 @@ export default Ember.Route.extend({
-                        postBodyParams: [
-                            {
-                                parameterName: "query",
-                                parameterPath: ["query", "bool", "must", 0, "query_string", "query"],
-                                defaultValue: "*"
-                            },
-                            {
-                                parameterName: "topic",
-                                parameterPath: ["query", "bool", "filter", 0, "term", "tags"]
-                            },
-                            {
-                                parameterName: "institution",
-                                parameterPath: ["query", "bool", "filter", 1, "term", "sources.raw"],
-                            },
-                            {
-                                parameterName: "scholar",
-                                parameterPath: ["query", "bool", "filter", 2, "term", "contributors.raw"],
-                            }
-                        ],
-                        facetDash: "scholar"
-                    },
-                ]
-            },
+          /* Each top level object is a dashboard view (i.e. a page with widgets) See first object for comments on properties */
             objectDetail: {
-                dasboardName: 'Object Detail Dashboard',
+                dashboardName: 'Object Detail Dashboard',
+                /* Each widget goes through widget-adapter that uses the settings and parameters */
                 widgets: [
                     {
-                        widgetType: "object-detail-widget",
+                        widgetType: "object-detail-widget",       // Corresponds to the Ember component name
                         background_color: "000000",
-                        name: "",
-                        width: 12,
-                        post_body: {},
-                        postBodyParams: [
-                            {
-                                parameterName: "id",
+                        name: "",       // Showing a name displays widget title, empty name omits title
+                        width: 12,       // Widgets are floated left and width is defined in grid columns 0-12
+                        post_body: {},       // Raw elastic query to be sent with the request for SHARE
+                        postBodyParams: [       // List of parameters to add to post_body. Params overrides post_body if there are same parameter paths
+                            {
+                                parameterName: "id",       // the name of the url parameter, the value of this parameter will be used as value of the parameterpath below
+                                /*  parameterPath: Path of the parameter object. Converts to:
+                                *   "query": {
+                                *       "bool" : {
+                                *           "must" : [
+                                *               0 : {
```

Comment:
Yes, I wanted to put that in there to show what 0 is, I thought omitting it would be confusing and may not indicate that it's an index. I can redo the output and add that as a comment there, what do you think?